app/components/remove-log-popup.js

```diff
@@ -17,7 +17,7 @@ export default Ember.Component.extend({
 
       return job.removeLog().then(() => {
         this.get('flashes').success('Log has been successfully removed.');
-      }).fail((xhr) => {
+      }, (xhr) => {
```

Comment:
yeah, I think explicit is better than “second argument represents failure”, I just thought [`catch`](http://emberjs.com/api/classes/Ember.PromiseProxyMixin.html) was the only option (or `finally`).